src/register/constants/pbaNumber.ts

```diff
@@ -19,43 +19,53 @@ export const pbaNumber = {
     ],
     groups: [
       {
-        input: {
-          label: {
-            text: 'PBA number 1 (optional)',
-            classes: 'govuk-label--m',
-          },
-          control: 'PBAnumber1',
-          validators: ['pbaNumberPattern', 'pbaNumberMaxLength', 'pbaNumberMinLength'],
-          validationError: {
-            value: 'Enter a valid PBA number',
-            controlId: 'PBAnumber1',
+        fieldset: [
+          {
+            legend: {
+              text: 'What\'s the Payment by Account (PBA) number for your organisation?',
+              classes: 'govuk-fieldset__legend--xl'
+            }
           },
-          classes: 'govuk-!-width-two-thirds',
-        },
-      },
-      {
-        input: {
-          label: {
-            text: 'PBA number 2 (optional)',
-            classes: 'govuk-label--m',
+          {
+            input: {
+              label: {
+                text: 'PBA number 1 (optional)',
+                classes: 'govuk-label--m',
+              },
+              control: 'PBAnumber1',
+              validators: ['pbaNumberPattern', 'pbaNumberMaxLength', 'pbaNumberMinLength'],
+              validationError: {
+                value: 'Enter a valid PBA number',
+                controlId: 'PBAnumber1',
+              },
+              classes: 'govuk-!-width-two-thirds',
+            },
           },
-          control: 'PBAnumber2',
-          validators: ['pbaNumberPattern', 'pbaNumberMaxLength', 'pbaNumberMinLength'],
-          validationError: {
-            value: 'Enter a valid PBA number',
-            controlId: 'PBAnumber2',
+          {
+            input: {
+              label: {
+                text: 'PBA number 2 (optional)',
+                classes: 'govuk-label--m',
+              },
+              control: 'PBAnumber2',
```

Comment:
Same thing about the naming convention: `'pbaNumber2'` (or `'PBANumber2'` if that's the pattern you've gone for, above.)

Reply:
changed to PBANumber2